packages/appstore/src/components/wallet-transfer/components/transfer-account-selector/transfer-account-selector.tsx

```diff
@@ -0,0 +1,214 @@
+
+type TTransferAccountSelectorProps = {
+    contentScrollHandler?: React.UIEventHandler<HTMLDivElement>;
+    is_wallet_name_visible?: boolean;
+    name: 'from_account' | 'to_account';
+    setIsWalletNameVisible?: (value: boolean) => void;
+};
+
+type TAccountSelectorTransferTileProps = {
+    is_mobile?: boolean;
+    label?: string;
+    selected_account?: TTransferAccount;
+    placeholder?: string;
+};
+
+const ChevronIcon = () => {
+    return (
+        <div className='transfer-account-selector__chevron-icon'>
+            <Icon icon='IcChevronDown' data_testid='dt_chevron_icon' />
+        </div>
+    );
+};
+
+const AccountSelectorTransferTile = ({
+    is_mobile,
+    label,
+    placeholder,
+    selected_account,
+}: TAccountSelectorTransferTileProps) => {
+    return (
+        <React.Fragment>
+            <div className='transfer-account-selector__content'>
+                <div className='transfer-account-selector__heading-with-chevron'>
+                    <div className='transfer-account-selector__heading'>
+                        <Text size={is_mobile ? 'xxs' : 'xs'}>{label}</Text>
+                    </div>
+
+                    {is_mobile && <ChevronIcon />}
+                </div>
+
+                {selected_account ? (
+                    <WalletTransferTile
+                        account={selected_account}
+                        className='transfer-account-selector__value'
+                        is_mobile={is_mobile}
+                    />
+                ) : (
+                    <Text size={is_mobile ? 'xxs' : 'xs'} weight='bold'>
+                        {placeholder}
+                    </Text>
+                )}
+            </div>
+
+            {!is_mobile && (
+                <React.Fragment>
+                    <WalletJurisdictionBadge
+                        is_demo={Boolean(selected_account?.is_demo)}
+                        shortcode={selected_account?.shortcode}
+                    />
+                    <ChevronIcon />
+                </React.Fragment>
+            )}
+        </React.Fragment>
+    );
+};
+
+const TransferAccountSelector = observer(
+    ({ contentScrollHandler, is_wallet_name_visible, name, setIsWalletNameVisible }: TTransferAccountSelectorProps) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+        const { values, setErrors, setFieldValue } = useFormikContext<TInitialValues>();
+        const { from_account, to_account } = values;
+        const { active_wallet, trading_accounts, wallet_accounts } = useTransferBetweenAccounts();
+        const [is_list_modal_open, setIsListModalOpen] = useState(false);
+        const active_wallet_name = getAccountName({ ...active_wallet });
+        const label = name === 'from_account' ? localize('Transfer from') : localize('Transfer to');
+        const placeholder =
+            name === 'from_account' || !to_account ? localize('Select a trading account or a Wallet') : '';
```

Comment:
Shouldn't we avoid using `localize` as it will soon be deprecated? Would it work it it were changed to something like:
```suggestion
        const label = (
        <Localize i18n_default_text={name === 'from_account' ? 'Transfer from' : 'Transfer to'} />
        );

        const placeholder = (
        <Localize i18n_default_text={name === 'from_account' || !to_account ? 'Select a trading account or a Wallet' : ''} />
        );
```

Reply:
@lubega-deriv 
you are right 👍

Reply:
[reference](https://deriv-group.slack.com/archives/C3AHMSM3P/p1677553396124379)

Reply:
Noted @heorhi-deriv thanks for the reference 👍